src/users/components/jurisdictions-access-options/jurisdiction-access-options.component.html

```diff
@@ -0,0 +1,45 @@
+<div [formGroup]="jurisdictions">
+  <ng-container *ngFor="
+          let jurisdiction of jurisdictions.controls;
+          let i = index
+        ">
+    <!-- The jurisdiction name
```

Comment:
This can be gone now?

Reply:
I guess if it's commented out it can go

Reply:
Removed in commit:
https://github.com/hmcts/rpx-xui-manage-organisations/commit/f13b896f6d2d616fe1b63f082089d9c50d05af97